src/internal/operators/retryWhen.ts

```diff
@@ -61,64 +62,37 @@ import { lift } from '../util/lift';
  */
 export function retryWhen<T>(notifier: (errors: Observable<any>) => Observable<any>): MonoTypeOperatorFunction<T> {
   return (source: Observable<T>) =>
-    lift(source, function (this: Subscriber<T>, source: Observable<T>) {
-      const subscriber = this;
+    wrappedLift(source, (subscriber: Subscriber<T>, source: Observable<T>) => {
       const subscription = new Subscription();
```

Comment:
issue: As with `repeatWhen`, it's not clear to me why this `Subscription` instance is necessary. Maybe I need a coffee?

Reply:
It's not. Just another one I didn't really touch as I was going through other that updating the `OperatorSubscriber`... will update.